core/src/main/java/com/linecorp/armeria/client/HttpChannelPool.java

```diff
@@ -515,7 +515,7 @@ private void notifyConnect(SessionProtocol desiredProtocol, PoolKey key, Future<
                     // Clean up old unhealthy channels by iterating from the beginning of the queue.
                     final Deque<PooledChannel> queue = getPool(protocol, key);
                     if (queue != null) {
-                        for (; ; ) {
+                        for (;;) {
```

Comment:
looks like this one can't be format automatically? IDEA setting has `space after semicolon`

Reply:
IntelliJ IDEA seems to have a bug :sweat_smile: